reframe/frontend/check_filters.py

```diff
@@ -1,32 +1,44 @@
+import re
+
 import reframe.core.runtime as rt
 import reframe.utility.sanity as util
 
 
-def have_name(names):
+def have_name(regexp_names):
     def _fn(c):
```

Comment:
Shouldn't this return a boolean indicating whether the name is contained in any of the regexes?

Reply:
The old version of this function returns a test name, I have implemented the same behavior for the new one.

Reply:
This: `return c.name in names` returns a boolean.

Reply:
Ooops, sorry I was wrong.

Reply:
Done.